crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -466,7 +466,7 @@ impl InspectorStack {
     }
 
     #[inline(always)]
-    fn as_mut(&mut self) -> InspectorStackRefMut<'_> {
+    pub fn as_mut(&mut self) -> InspectorStackRefMut<'_> {
```

Comment:
I've thought about this, it could be slower to use `as_mut` because then field accesses have to go through an extra layer of indirection (&mut RefMut -> &mut Inner -> field), whereas passing `InspectorStack` doesn't; it has to be measured

Reply:
current inspector impls for `InspectorStack` already delegate to `as_mut()`, so I think this is already happening?

Reply:
They would get inlined which removes the indirection

Reply:
yeah, this indeed results in performance decrease

changed back in https://github.com/foundry-rs/foundry/pull/9011/commits/56eac96631d2e1a318953197b4851bf2ba852295 and added `with_stack` helper which constructs a temp `InspectorStack` by moving out of `InspectorRefMut`

I'll likely clean this up with `--isolate` refactor I am doing next